packages/panzer/dof-mgr/src/Panzer_UniqueGlobalIndexer.hpp

```diff
@@ -299,8 +299,8 @@ class UniqueGlobalIndexer : public UniqueGlobalIndexerBase {
    /** Access the local IDs for an element. The local ordering is according to
      * the <code>getOwnedAndGhostedIndices</code> method.
      */
-   const std::vector<LocalOrdinalT> & getElementLIDs(LocalOrdinalT localElmtId) const
-   { return localIDs_[localElmtId]; }
+   const Kokkos::View<const LocalOrdinalT*,PHX::Device> getElementLIDs(LocalOrdinalT localElmtId) const
+    { return Kokkos::subview(localIDs_k_, localElmtId, Kokkos::ALL() ); }
 
    /** Access the local IDs for an element. The local ordering is according to
      * the <code>getOwnedAndGhostedIndices</code> method. Note
```

Comment:
Given these changes, is there still a need for the other `getElementLIDs()` routine down on lines 305&ndash;317?

Reply:
I look at the later one as give me a bunch of LIDs for a bunch of elems.